src/tags/models.py

```diff
@@ -8,16 +8,22 @@
 
 
 def total_xp_by_tags(user, tags):
-    """ 
-        Returns user's total xp over active_semester filtered by tags
-        Args:
-            tags - a queryset or list of Tag objects
-    """ 
+    """
+    Returns user's total xp over active_semester filtered by tags
+
+    Args:
+        tags (list[str], Queryset<Tag>): a queryset or list of Tag objects
```

Comment:
Is the ability to accept a Queryset actually tested?  if not, can you add one?

Reply:
added test for qs and list